silkrpc/core/evm_debug.hpp

```diff
@@ -121,7 +121,16 @@ struct DebugTrace {
     DebugConfig debug_config;
 };
 
+struct DebugTraceResult {
+    DebugTrace debug_trace;
+};
+
+struct DebugTraceResultList {
+    std::vector<DebugTrace> debug_traces;
+};
+
 void to_json(nlohmann::json& json, const DebugTrace& debug_trace);
+void to_json(nlohmann::json& json, const DebugTraceResult& debug_trace_result);
```

Comment:
This will be removed using the new approach and `to_json` for `DebugTraceResultList ` will be added

Reply:
ok